_data/changelogs/tooltip.yml

```diff
@@ -19,4 +19,31 @@ changelogURL:
   #   githubPr:
   #   githubRepo:
   #   versionUswds:
-# end changelog item
\ No newline at end of file
+
+items:
+  # 3.2.0 release
+  - date: 2022-10-19
+    summary: Tooltip allows for dynamic initialization.
+    summaryAdditional: Tooltip initializes on first interaction and checks if component has been initialized before toggling. If it hasn't initialized properly it will call a setup attributes function.
+    affectsAccessibility: false
+    affectsAssets: false
+    affectsGuidance: false
+    affectsJavascript: true
+    affectsMarkup: false
+    affectsStyles: true
+    githubPr: 4955
+    githubRepo: uswds
+    versionUswds: 3.2.0
+  # 3.0 release
+  - date: 2022-04-28
+    summary: USWDS 3.0.0 released.
+    summaryAdditional: USWDS 3.0 allows teams to update their projects to modern Sass syntax, improve performance, and reduce the size of project CSS.
+    affectsAccessibility: true
+    affectsAssets: true
+    affectsGuidance: true
+    affectsJavascript: true
+    affectsMarkup: true
+    affectsStyles: true
+    githubPr: 4656
+    githubRepo: uswds
+    versionUswds: 3.0.0
```

Comment:
Found a couple extra items to include: 
```
  - date: 2021-11-01
    summary: Added automatic sanitizing.
    summaryAdditional: The design system now automatically sanitizes content in elements we compose with JavaScript. This means that components like Combobox, Tooltip, File Input, and Date Picker will sanitize any content passed to them. This helps protect any design system implementation against malicious XSS attacks through these components.
    affectsJavascript: true
    githubPr: 4329
    githubRepo: uswds
    versionUswds: 2.12.2
  - date: 2021-03-17
    summary: Fixed tooltip positioning. 
    summaryAdditional: We fixed an issue that prevented some longer tooltips in some positions to display incorrectly.
    affectsJavascript: true
    affectsStyles: true
    githubPr: 4062
    githubRepo: uswds
    versionUswds: 2.11.0
```

Reply:
Updated in 122451ca14e25136a52f4a2bae13694d86bb2ff4, great notes.